theseus/extlib/cusolver_lu_solver.cpp

```diff
@@ -0,0 +1,334 @@
+
```

Comment:
Please add the license lines at the start of this and all new files created in this PR.
```
# Copyright (c) Meta Platforms, Inc. and affiliates.
#
# This source code is licensed under the MIT license found in the
# LICENSE file in the root directory of this source tree.
```